server/cherry_pick.go

```diff
@@ -108,7 +108,7 @@ func (s *Server) doCherryPick(ctx context.Context, version string, milestoneNumb
 	}
 	releaseBranch := fmt.Sprintf("upstream/%s", version)
 	repoFolder := fmt.Sprintf("/home/ubuntu/git/mattermost/%s", pr.RepoName)
-	cmd := exec.Command("/home/ubuntu/git/devops/cherry-pick.sh", releaseBranch, strconv.Itoa(pr.Number), pr.MergeCommitSHA)
+	cmd := exec.Command("/app/scripts/cherry-pick.sh", releaseBranch, strconv.Itoa(pr.Number), pr.MergeCommitSHA)
```

Comment:
We seriously need to move from this hardcoded path to something that is coming from the config. @ashishbhate - makes sense to track that in a new ticket?

Reply:
Anything config, I completely agree, but this is the underlying foundation of the image and I don't think this is needed to get parameterized or put in config.json.

Reply:
Yep, I made a ticket yesterday when trying to get mattermod working on my local machine. https://mattermost.atlassian.net/browse/MM-27192